spring-cloud-zuul-ratelimit-core/src/main/java/com/marcosbarbero/cloud/autoconfigure/zuul/ratelimit/support/RateLimitConstants.java

```diff
@@ -16,26 +16,22 @@
 
 package com.marcosbarbero.cloud.autoconfigure.zuul.ratelimit.support;
 
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+
 /**
  * @author Marcos Barbero
+ * @author Liel Chayoun
  * @since 2018-04-05
  */
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
 public final class RateLimitConstants {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1118)